src/EditorFeatures/Core.Wpf/InlineParamNameHints/InlineParamNameHintsTaggerProvider.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.ComponentModel.Composition;
+using Microsoft.VisualStudio.Text;
+using Microsoft.VisualStudio.Text.Editor;
+using Microsoft.VisualStudio.Text.Tagging;
+using Microsoft.VisualStudio.Utilities;
+
+namespace Microsoft.CodeAnalysis.Editor.InlineParamNameHints
+{
+    /// <summary>
+    /// The provider that is used as a middleman to create the tagger so that the data tag
+    /// can be used to create the UI tag
+    /// </summary>
+    [Export(typeof(ITaggerProvider))]
+    [ContentType("csharp")]
```

Comment:
this seems wrong. @jasonmalinowski ?

Reply:
Yep we've got a comment for this:

https://github.com/dotnet/roslyn/blob/2f626aa36002c7c6803c123534d1bebf8e5fbd86/src/EditorFeatures/Text/ContentTypeNames.cs#L9

More generally at some point we'll probably switch this to the Roslyn content type once we're doing VB too but let's start it here for now.